metabase/metabase
# Add setup tracking

Epic: https://github.com/metabase/metabase/issues/18291

Adds setup tracking events. 

How to test:
- Change `isProduction` to `true`
- Install Snowplow Debugger extension
- Run a fresh Metabase instance locally and complete the setup flow
- Make sure that `step_seen`, `database_selected`, `add_data_later_clicked` work
